assets/sass/components/global/_googlesitekit-data-block.scss

```diff
@@ -185,6 +207,12 @@
 		padding-bottom: $grid-gap-desktop;
 		padding-top: $grid-gap-desktop + 4px;
 	}
+
+	/* stylelint-disable no-descending-specificity */
```

Comment:
I'm guessing this exception is necessary but it's not obvious why – would you please add a comment to explain? E.g. https://github.com/google/site-kit-wp/blob/de906c7c097d9e1e1f9f274365bb51da2900dac5/assets/js/modules/analytics-4/datastore/report.js#L334

Reply:
Thanks for calling this out - I've had another look and was able to reorganise the CSS rules to avoid needing to disable this stylelint rule.